setup.cfg

```
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E111,E114
+ignore = E111,E114,E902
```

not sure if we'd want to describe all ignored errors here since you can find the commit which added it and see
https://github.com/zidarsk8/ggrc-core/commit/203f6cac925b23fa6d6c9ce947874a543420510a